editor/plugins/curve_editor_plugin.cpp

```diff
@@ -782,12 +782,13 @@ bool CurvePreviewGenerator::handles(const String &p_type) const {
 	return p_type == "Curve";
 }
 
-Ref<Texture> CurvePreviewGenerator::generate(const Ref<Resource> &p_from) {
+Ref<Texture> CurvePreviewGenerator::generate(const Ref<Resource> &p_from, const Size2 p_size) const {
 
 	Ref<Curve> curve_ref = p_from;
 	ERR_FAIL_COND_V(curve_ref.is_null(), Ref<Texture>());
 	Curve &curve = **curve_ref;
 
+	// FIXME: Should be ported to use p_size as done in b2633a97
```

Comment:
@groud This one was missed in b2633a97 so it's probably buggy now.

Reply:
(While at it, might be worth grepping for `file_dialog/thumbnail_size` and checking if other places need to be fixed - I see at least the EditorPreviewGenerator docs that mention it.)

Reply:
Thanks :)